src/classes/issues-processor.ts

```diff
@@ -804,34 +836,14 @@ export class IssuesProcessor {
     }
   }
 
-  private async _getPullRequest(
-    issue: Issue
-  ): Promise<IPullRequest | undefined | void> {
-    const issueLogger: IssueLogger = new IssueLogger(issue);
-
-    try {
-      this._consumeIssueOperation(issue);
-      this._statistics?.incrementFetchedPullRequestsCount();
-
-      const pullRequest = await this.client.pulls.get({
-        owner: context.repo.owner,
-        repo: context.repo.repo,
-        pull_number: issue.number
-      });
-
-      return pullRequest.data;
-    } catch (error) {
-      issueLogger.error(`Error when getting this $$type: ${error.message}`);
-    }
-  }
-
   // Delete the branch on closed pull request
   private async _deleteBranch(issue: Issue): Promise<void> {
     const issueLogger: IssueLogger = new IssueLogger(issue);
 
     issueLogger.info(`Delete branch from closed $$type - ${issue.title}`);
 
-    const pullRequest = await this._getPullRequest(issue);
+    const pullRequest: IPullRequest | undefined | void =
```

Comment:
I don't think you have to specify the type here. Could just do `const pullRequest = await this._getPullRequest(issue)`

Reply:
FYI: I do this on purpose because I prefer explicit types; easier to review and code IMO